app/src/main/res/drawable/ic_info_icon_gray_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#999999"
+        android:viewportHeight="24.0" android:viewportWidth="24.0"
+        android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="#FF000000" android:pathData="M11,17h2v-6h-2v6zM12,2C6.48,2 2,6.48 2,12s4.48,10 10,10 10,-4.48 10,-10S17.52,2 12,2zM12,20c-4.41,0 -8,-3.59 -8,-8s3.59,-8 8,-8 8,3.59 8,8 -3.59,8 -8,8zM11,9h2L13,7h-2v2z"/>
+</vector>
```

Comment:
EOF is missing, add one blank line at the end.

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done